modules/shorten.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file shorten.js
+ * @description Module for shortening links using "acmurl" (Rebrandly API).
+ * @author Storm_FireFox1
+ */
+
+global.config = require('../config.json')
```

Comment:
delete

Reply:
All configuration is now done by environment variables, check 4219eda.